AI_Graphics/AI_Graphics/PostProcessingManager.cs

```diff
@@ -9,9 +9,19 @@ namespace AIGraphics
     internal class PostProcessingManager : MonoBehaviour
     {
         private string _lensDirtTexturePath;
-        internal List<string> LensDirtPaths { get; private set; }
+        // Texture Lists don't have to be instanced.
+        internal static List<string> LensDirtPaths { get; private set; }
+        internal static List<Texture2D> LensDirts { get; private set; }
         internal List<Texture2D> LensDirtPreviews { get; private set; }
```

Comment:
any reason why previews are left as instanced?

Reply:
Well I think I forgot to make it static

Reply:
if you plan to change it in a subsequent change, please resolve convo, then I'll approve so we can merge this, thx